recipes/thriftpy/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: thriftpy
+  version: "0.3.5"
+
+source:
+  git_url: https://github.com/eleme/thriftpy
+  git_rev: v0.3.5
+
+build:
+  number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
+  string: py{{ environ.get('PY_VER').replace('.', '') }}_{{ environ.get('GIT_BUILD_STR', 'GIT_STUB') }}
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
Every packages that uses `setuptools` you use `python setup.py install --single-version-externally-managed --record record.txt` to install instead of just `python setup.py install `.